### DISCUSSION TITLE
Two Pricing pages is confusing

### DISCUSSION THREAD

**Discussion by fergusmeiklejohn** [supabase/supabase#1794] [Questions] [answered]
ATM we have two pricing pages:
https://supabase.io/pricing
https://supabase.io/docs/pricing

I think that's confusing and it would be better to name the second (docs) pricing page and link "Sponsor"

**Comment by inian** [comment, marked as answer]
Hi @fergusmeiklejohn, https://supabase.io/docs/pricing redirects me to https://supabase.io/pricing 
Is this not happening for you?

**Reply by silentworks** [reply]
I get redirected too.

**Reply by fergusmeiklejohn** [reply]
If I click on `pricing` under `See Also` I get  https://supabase.io/docs/pricing then if I refresh the page I'm redirected to https://supabase.io/pricing
![Screenshot 2021-06-03 at 11 57 07](https://user-images.githubusercontent.com/11864025/120584715-0d5df580-c463-11eb-9c65-567837dd16a0.png)

**Reply by kiwicopple** [reply]
Thanks for pointing this out @fergusmeiklejohn ! 

Removed the docs page to remove any confusion:
https://github.com/supabase/supabase/pull/1807
